WordPress/WordPressStatsWidgets/StatsWidgets.swift

```diff
@@ -7,5 +7,10 @@ struct WordPressStatsWidgets: WidgetBundle {
         WordPressHomeWidgetToday()
         WordPressHomeWidgetThisWeek()
         WordPressHomeWidgetAllTime()
+#if JETPACK_STATS_WIDGETS
+        if #available(iOS 16.0, *) {
```

Comment:
As mentioned above, something like this could be used to avoid creating a configuration

```suggestion
        if AppConfiguration.isJetpack, #available(iOS 16.0, *)  {
```

Reply:
That's great, better than my existing solution 👍.